bqrole/dataset.go

```diff
@@ -84,7 +84,8 @@ func PermitDataset(role bq.AccessRole, project string, users, datasets []string)
 
 // grantBQRole grants user roles/bigquery permission
 func grantBQRole(project, user string, role string) error {
-	policy, err := fetchCurrentPolicy(project)
+	policy, err := FetchCurrentPolicy(project)
+
```

Comment:
[nits] IMO, this blank line is unnecessary (not consistent with [here](https://github.com/hirosassa/bqiam/pull/32/files#diff-d6cbf47806cb27b6da400d2a751f05527f57d2c49ddc7f1727ccbab554191a63R62))